drip/drips.py

```diff
@@ -84,7 +86,8 @@ def plain(self):
     def get_from_(self):
         if self.drip_base.from_email_name:
             from_ = "%s <%s>" % (
-                self.drip_base.from_email_name, self.drip_base.from_email)
+                self.drip_base.from_email_name, self.drip_base.from_email
+            )
```

Comment:
we should use a string template, or at least `"{} <{}>".format()` for this

Reply:
I think we should change this in other PR where we check all the format strings